paasta_tools/utils.py

```diff
@@ -551,8 +551,10 @@ def get_env_dictionary(self) -> Dict[str, str]:
         }
         if self.get_gpus() is not None:
             env["PAASTA_RESOURCE_GPUS"] = str(self.get_gpus())
-        if self.get_docker_image():
+        try:
             env["PAASTA_GIT_SHA"] = get_git_sha_from_dockerurl(self.get_docker_url())
+        except Exception:
```

Comment:
is there a more specific exception that you can catch?

Reply:
No. This is only to work around the failed travis test, where git_registry doesn't exist. Docker image could be missing as well in some test cases. 
This is not intended to catch any errors in practice. And we don't really need to catch the error if we can't get dockerurl. The service wouldn't even run.